contracts/pcv/IPCVDepositAggregator.sol

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./IPCVDeposit.sol";
+import "../external/Decimal.sol";
+
+/** 
+ @title a PCV Deposit aggregation interface
+ @author Fei Protocol
+
+ This contract is a single interface for allocating a specific token to multiple PCV Deposits.
+ The aggregator handles new incoming funds and outgoing funds by selecting deposits which are over or under-funded to save for gas and efficiency
+*/
+
+interface IPCVDepositAggregator {
```

Comment:
unused, we should remove unless it got audited and we want to use it in the future

Reply:
done